docs/how-to-guides/backup-rook-ceph-volumes.md

```diff
@@ -0,0 +1,235 @@
+# Backup Rook Ceph Volume on S3 using Velero
+
+## Contents
+
+- [Introduction](#introduction)
+- [Learning objectives](#learning-objectives)
+- [Prerequisites](#prerequisites)
+- [Steps](#steps)
+  - [Step 1: Deploy Velero](#step-1-deploy-velero)
+    - [Config](#config)
+    - [Deploy](#deploy)
+  - [Step 2: Deploy sample workload](#step-2-deploy-sample-workload)
+  - [Step 3: Annotate pods](#step-3-annotate-pods)
+  - [Step 4: Backup entire namespace](#step-4-backup-entire-namespace)
+  - [Step 5: Restore in another cluster](#step-5-restore-in-another-cluster)
+- [Additional resources](#additional-resources)
+
+## Introduction
+
+Rook is a component of Lokomotive which provides storage on Packet cloud. Taking regular backup of
```

Comment:
Should we cross-link here to rook documentation?

Reply:
Added.